src/game/time_attack/results.c

```diff
@@ -50,7 +50,7 @@ u32 CreateTimeAttackResultsCutScene(u32 finishTime)
     u8 i;
     u8 isBossLevel;
     u8 level;
-    gLoadedSaveGame->score += gRingCount;
+    gLoadedSaveGame->score += (s16)gRingCount;
```

Comment:
Makes sense (well doesn't make sense but ringCount as unsigned global is more obvious

Reply:
Yeah, that's what I was thinking!
And it's better to cast the odd case than the common case!